node/src/BaseClient.ts

```diff
@@ -433,9 +433,9 @@ export class BaseClient {
             const pointer = message.respPointer;
 
             if (typeof pointer === "number") {
-                resolve(valueFromSplitPointer(0, pointer));
+                resolve(valueFromSplitPointer(0, pointer, true));
```

Comment:
please add TODO commands to revert these under https://github.com/valkey-io/valkey-glide/pull/1953